pkg/cmd/admin/prune/jobs.go

```diff
@@ -0,0 +1,179 @@
+package prune
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"text/tabwriter"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/apis/batch"
+	kclient "k8s.io/kubernetes/pkg/client/unversioned"
+	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+	"github.com/openshift/origin/pkg/job/prune"
+)
+
+const PruneJobsRecommendedName = "jobs"
+
+const (
+	jobsLongDesc = `Prune old completed and failed jobs
+
+By default, the prune operation performs a dry run making no changes to the jobs.
+A --confirm flag is needed for changes to be effective.
+`
+
+	jobsExample = `  # Dry run deleting all but the last completed job
+  %[1]s %[2]s --keep-complete=1
+
+  # To actually perform the prune operation, the confirm flag must be appended
+  %[1]s %[2]s --keep-complete=1 --confirm`
+)
+
+// PruneJobsOptions holds all the required options for pruning jobs.
+type PruneJobsOptions struct {
+	Confirm         bool
+	KeepYoungerThan time.Duration
+	KeepComplete    int
+	KeepFailed      int
+	Namespace       string
+
+	KClient kclient.Interface
+	Out     io.Writer
+}
+
+// NewCmdPruneJobs implements the OpenShift cli prune jobs command.
+func NewCmdPruneJobs(f *clientcmd.Factory, parentName, name string, out io.Writer) *cobra.Command {
+	opts := &PruneJobsOptions{
+		Confirm:         false,
+		KeepYoungerThan: 60 * time.Minute,
+		KeepComplete:    5,
+		KeepFailed:      1,
+	}
+
+	cmd := &cobra.Command{
+		Use:        name,
+		Short:      "Remove old completed and failed jobs",
+		Long:       jobsLongDesc,
+		Example:    fmt.Sprintf(jobsExample, parentName, name),
+		SuggestFor: []string{"job", "jobs"},
+		Run: func(cmd *cobra.Command, args []string) {
+			kcmdutil.CheckErr(opts.Complete(f, cmd, args, out))
+			kcmdutil.CheckErr(opts.Validate())
```

Comment:
Good catch, we don't do that in other prune commands either, will fix there as well.